.travis.yml

```diff
@@ -5,7 +5,7 @@
 sudo: false
 language: node_js
 node_js:
-    - '0.12'
+    - 'node'
```

Comment:
![screen shot 2015-10-19 at 9 37 44 am](https://cloud.githubusercontent.com/assets/883126/10584051/10360bac-7645-11e5-8589-63de11eb4199.png)


They mean the same thing 
http://docs.travis-ci.com/user/languages/javascript-with-nodejs/